packages/create-app/src/index.ts

```diff
@@ -26,10 +26,11 @@ import { version } from '../package.json';
 import createApp from './createApp';
 
 const main = (argv: string[]) => {
-  program.name('backstage-create-app').version(version);
-
   program
-    .description('Creates a new app in a new directory')
+    .name('backstage-create-app')
+    .version(version)
+    .description('Creates a new app in a new directory or specified [path]')
+    .usage('[path] [options]')
```

Comment:
Hi @OrkoHunter ! I updated the description, and leveraged the `usage` method to include instructions on the `[path]` argument.

I made a comment on the issue itself explaining that I originally tried to use the `arguments` method, but that seemed to require me to store the argument in an external variable which I would then pass to the `createApp` function. This would've required a bit more of a refactor, but if you think that would be better, please let me know! Thanks for the help!

Reply:
You should be able to use `arguments`. https://www.npmjs.com/package/commander#command-arguments and then it will be passed to the `createApp` function here: https://github.com/backstage/backstage/blob/a88073c29afb0f1e9b591f8c5a0f2e1e992075b9/packages/create-app/src/createApp.ts#L88

Reply:
` .argument('[path]', 'path to create the app', null)` or something should work I think